src/sentry/migrations/0112_add_repositoryprojectpathconfig.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-10-14 18:24
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+import sentry.db.models.fields.bounded
+import sentry.db.models.fields.foreignkey
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    atomic = True
+
+
+    dependencies = [
+        ('sentry', '0111_snuba_query_event_type'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='RepositoryProjectPathConfig',
+            fields=[
+                ('id', sentry.db.models.fields.bounded.BoundedBigAutoField(primary_key=True, serialize=False)),
+                ('date_updated', models.DateTimeField(default=django.utils.timezone.now)),
+                ('date_added', models.DateTimeField(default=django.utils.timezone.now, null=True)),
+                ('input_path', models.TextField()),
+                ('output_path', models.TextField()),
```

Comment:
stack_path and repository_path might be clearer names for how these columns would be used.

Reply:
@scefali what are your thoughts? I like these names

Reply:
@MeredithAnya maybe `base_stack_path` and `base_repository_path` might make more sense

Reply:
what about `stack_root`, `source_root`?

Reply:
@MeredithAnya sounds good to me

Reply:
stack_root an source_root are 🌈